packages/meteor-autoform-ufs/lib/client/af-file-upload.js

```diff
@@ -15,6 +15,7 @@ import Compress from 'compress.js';
 const compress = new Compress();
 
 export function isImage(file) {
+  if (!file) return false;
```

Comment:
Mikor tud a file undefined lenni?

Reply:
Ez nagyon jó kérdés. Kivizsgáltuk, és átírjuk a megfelelő helyre a feltételt, hogy ne fordulhasson ilyen elő. (Feltöltés közben már bekerül a shareddocsba a file, csak még nincs neki path-ja, amit itt a file-ként adnánk be.)